runtime/onert/core/include/ir/Graph.h

```diff
@@ -172,13 +172,13 @@ class Graph
   OperandIndexSequence _outputs;
   std::unordered_map<std::string, IOIndex> _name_to_input;
   std::unordered_map<std::string, IOIndex> _name_to_output;
-  // Child subgraphs
-  std::shared_ptr<Subgraphs> _subgraphs;
+  // model for child subgraphs
+  std::shared_ptr<Model> _model;
```

Comment:
Hmm. IMHO, this field may not used any more. It may be used before `SubGraphs`(`Model` in this PR) is introduced. 
@ragmani Do you know about this?

Reply:
@hseok-oh Thanks for comment. I also have same question about this field. That said, what about removing this line in another PR when it turned out unnecessary? It is beyond this PR (mechanical renaming).

Reply:
Right. And I confirmed that cpu backend and interpreter test are passed when I removed this field and related code.

Reply:
I can't remember exactly. If there is another way to find child subgraphs without this field and this field is not used anymore, you can remove this field.

Reply:
Let's check on #9308